### PR TITLE
Better error messages #549

### PR DESCRIPTION
**Pull request by banasiakadam60**

### CODE REVIEW

**Comment by magdasta** [comment]
Are we sure errors 246 and 247 are result of too low RAM limit?

**Comment by banasiakadam60** [comment]
I found it here: https://github.com/imapp-pl/golem/wiki/Common-problems

File: gnr/gnrapplicationlogic.py [hunk 1]
```diff
@@ -518,13 +518,21 @@ def test_task_computation_success(self, results, est_mem):
...
         self.customizer.gui.setEnabled('new_task', True)  # enable everything on 'new task' tab
         if self.customizer.new_task_dialog_customizer:
             self.customizer.new_task_dialog_customizer.test_task_computation_finished(False, 0)
 
+    @staticmethod
+    def __parse_error_message(error_msg):
+        if not error_msg:
+            return u"Unknown error"
+        if any(code in error_msg for code in ['246', '247', '500']):
+            return u"RAM limit is too low. Consider increasing max memory usage"
```
Comment (badb) [inline gnr/gnrapplicationlogic.py]: Maybe better solution is to make it: error_message + "There is a chance that you RAM limit..."

**Comment by magdasta** [comment]
Then looks fine for me.

**Comment by banasiakadam60** [comment]
@badb OK, I'll change it

**Pull request merged**